plugins/jira/skills/status-analysis/scripts/summarize_issue.py

```
@@ -0,0 +1,316 @@
+
+if __name__ == "__main__":
+    args = sys.argv[1:]
+
+    # Parse flags
+    only_significant, args = parse_flag(args, "--only-significant")
+    date_dir, args = parse_option(args, "--date-dir")
+    label_filter, args = parse_option(args, "--label")
+
+    if not args:
+        print(f"Usage: {sys.argv[0]} <ISSUE-KEY|issue.json|issues-dir/> [...]  [--date-dir DIR] [--only-significant] [--label LABEL]", file=sys.stderr)
+        print(f"Examples:", file=sys.stderr)
```

Ruff (0.15.13)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)